doc/usermanual/lighttable/panels/styles.xml

```diff
@@ -87,6 +93,10 @@
         overwriting the existing one; you need to provide a new unique style name in this case.
       </para>
 
+      <para>
+        See <i>create</i> how to create a hierarchical view.
```

Comment:
And \<i\> is not supported, you need to use \<emphasis\>.

Reply:
This should fix the travis check.

Reply:
Fixed, lets see.